.project-creation/.skeleton/src/main.py

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) 2023 Contributors to the Eclipse Foundation
```

Comment:
I think, if we move over existing code from older files we should keep the original year range ...
```suggestion
# Copyright (c) 2022-2023 Contributors to the Eclipse Foundation
```

Reply:
the main.py from skeleton app is new though.

Reply:
Yes - file is new, but content is "old"

Reply:
Updated